release/air_tests/air_benchmarks/mlperf-train/file_size_benchmark.sh

```diff
@@ -3,7 +3,7 @@
 
 set -x
 
-NUM_IMAGES_PER_FILE="32 512 2048"
+NUM_IMAGES_PER_FILE="32 512 8192"
```

Comment:
FYI we'll likely have to rerun tests once Clarence's changes are in https://github.com/ray-project/ray/pull/29679/files#diff-61b7d0ee907414af24c086ed52e41d3896eddff5b0807b2eaa46a683205db486R6